monitor/ConsoleReporter.js

```diff
@@ -72,16 +72,20 @@ define(function(require) {
 				// IE8 has console.log and JSON, so we can make a
 				// reasonably useful warn() from those.
 				// Credit to webpro (https://github.com/webpro) for this idea
-				if (typeof localConsole.log ==='function'
-					&& typeof JSON !== 'undefined') {
-					log = warn = function (x) {
-						if(typeof x !== 'string') {
+				// typeof localConsole.log will return 'object' in IE8, so can't test it with === 'function'
+				// Since this is more of a corner case for IE8, I'm ok to check it with !== 'undefined' to reduce complexity
+				if (typeof localConsole.log !== 'undefined' && typeof JSON !== 'undefined') {
```

Comment:
@sompylasar Thanks for the explanation.  It's def a concerning behavior, but I also haven't found any evidence of it after quite a few google searches.

I think the best thing to do is to proceed with this PR as is, and if/when we find evidence that need to be more paranoid and add `try/catch`, we can add it at that point.